lib/proxyquire.js

```diff
@@ -129,17 +130,47 @@ Proxyquire.prototype.load = function (request, stubs) {
   return this._withoutCache(this._parent, stubs, request, this._parent.require.bind(this._parent, request))
 }
 
+// Resolves a stub relative to a module.  Note that this only resolves
+// "relative" paths (e.g. "./foo") but not non-relative modules (e.g. "react").
+// Also, if noPreserveCache(), then we won't throw if the module can't
+// be resolved.
+Proxyquire.prototype._resolveRelativePath = function (path, moduleFilename) {
```

Comment:
`_resolvePath`, it may only be doing stuff with relative paths but it's responsible for parsing _all_ stub paths.

Reply:
Oh, good catch.  :)

Reply:
The comment here is now wildly incorrect, too.  :)